### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Sofinaye**
I have Added:

- Contact form consists of

-      Input fields of :

-       Full name 

-        Email address

- Textarea

- Footer social media Icons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -190,6 +190,26 @@ <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
...
+    <section class="contact">
+      <h3 class="contact-heading">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </h3>
+      <form action="https://formspree.io/f/xgebvavj" method="POST" class="contact-form">
+        <input type="text" class="contact-input" placeholder="Full name" maxlength="30" required>
+        <input type="email" class="contact-input" placeholder="Email address" required>
+        <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Enter text here"></textarea>
+        <button class="contact-btn" type="submit">Get in touch</button>
+      </form>
```
Comment (mahbubul14) [inline index.html:202]: - [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template. Kindly work on it. :+1: 

button_interaction

File: index.html [hunk 2]
```diff
@@ -190,6 +190,26 @@ <h3 class="qualification-heading"><i class="fa-solid fa-circle"></i>Skills</h3>
...
+    </section>
+    <footer class="footer">
+      <ul class="footer-social">
+        <li class="footer-social-item"><i class="fa-brands fa-github"></i></li>
+        <li class="footer-social-item"><i class="fa-solid fa-hand-peace"></i></li>
+        <li class="footer-social-item"><i class="fa-brands fa-twitter"></i></li>
+        <li class="footer-social-item"><i class="fa-brands fa-linkedin-in"></i></li>
+        <li class="footer-social-item"><i class="fa-solid fa-m"></i></li>
+      </ul>
+    </footer>
```
Comment (mahbubul14) [inline index.html:212]: - [ ] There is a line under the social media icon in the Figma template. You should do it on your design also. Kindly try to match your design to the Figma template. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
yourdesign | figma

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
**Hi @Team,**

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by binhussen — APPROVED** [review]
Hi @Sofinaye , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**